packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/utils/getDisplayedFilters.js

```diff
@@ -13,6 +13,26 @@ const customOperators = [
 ];
 
 const getDisplayedFilters = ({ formatMessage, users }) => {
+  const getDisplaynameFromUser = (user) => {
+    if (user.username) {
+      return user.username;
+    }
+    if (user.firstname && user.lastname) {
```

Comment:
Why AND and not OR?

Reply:
Yeah I can follow your arguments ... Let's keep it for now. What do you think of bringing this to the next FE sync? I think we could introduce a global utility function that format's users bases on attributes in a consistent way.

Reply:
might be useful to write this a comment imo if Gustav wasn't sure first time reading, there's a chance someone else might not in the future.